ui/api/src/main/java/org/jboss/forge/addon/ui/input/SelectComponent.java

```diff
@@ -30,4 +31,9 @@
    Converter<VALUETYPE, String> getItemLabelConverter();
 
    IMPLTYPE setItemLabelConverter(Converter<VALUETYPE, String> converter);
+
+   Converter<VALUETYPE, String> getInteractionItemLabelConverter();
+
```

Comment:
"Maybe we should change the UIContext.getProvider().isGUI() to return true when interaction mode is enabled"
:+1: